$MirOS: ports/www/firesomething/patches/patch-configure_in,v 1.8 2008/11/14 00:47:21 tg Exp $
$OpenBSD: patch-configure_in,v 1.5 2005/04/09 08:11:19 wilfried Exp $
--- configure.in.orig	Tue Oct 21 04:08:46 2008
+++ configure.in	Tue Dec  2 19:03:03 2008
@@ -62,7 +62,7 @@ dnl ====================================
 
 AC_PREREQ(2.13)
 AC_INIT(config/config.mk)
-AC_CONFIG_AUX_DIR(${srcdir}/build/autoconf)
+AC_CONFIG_AUX_DIR(build/autoconf)
 AC_CANONICAL_SYSTEM
 TARGET_CPU="${target_cpu}"
 TARGET_VENDOR="${target_vendor}"
@@ -870,7 +870,7 @@ tools are selected during the Xcode/Deve
                  int main() { return 0; }],
    result=yes,
    result=no)
-  AC_LANG_RESTORE
+  AC_LANG_C
   AC_MSG_RESULT($result)
 
   if test "$result" = "no" ; then
@@ -1558,10 +1558,10 @@ dnl Get mozilla version from central mil
 MOZILLA_VERSION=`$PERL $srcdir/config/milestone.pl -topsrcdir $srcdir`
 
 dnl Get version of various core apps from the version files.
-FIREFOX_VERSION=`cat $topsrcdir/browser/config/version.txt`
-THUNDERBIRD_VERSION=`cat $topsrcdir/mail/config/version.txt`
-SUNBIRD_VERSION=`cat $topsrcdir/calendar/sunbird/config/version.txt`
-SEAMONKEY_VERSION=`cat $topsrcdir/suite/config/version.txt`
+FIREFOX_VERSION=`cat ./browser/config/version.txt`
+THUNDERBIRD_VERSION=`cat ./mail/config/version.txt`
+SUNBIRD_VERSION=`cat ./calendar/sunbird/config/version.txt`
+SEAMONKEY_VERSION=`cat ./suite/config/version.txt`
 
 AC_DEFINE_UNQUOTED(MOZILLA_VERSION,"$MOZILLA_VERSION")
 AC_DEFINE_UNQUOTED(MOZILLA_VERSION_U,$MOZILLA_VERSION)
@@ -2222,12 +2222,22 @@ case "$target" in
 	esac
 	;;
 
-*-openbsd*)
-    DLL_SUFFIX=".so.1.0"
+*-midnightbsd*)
+    DLL_SUFFIX=".so"
     DSO_CFLAGS=''
     DSO_PIC_CFLAGS='-fPIC'
     DSO_LDOPTS='-shared -fPIC'
     if test "$LIBRUNPATH"; then
+       DSO_LDOPTS="-R$LIBRUNPATH $DSO_LDOPTS"
+    fi
+    ;;
+
+*-openbsd*|*-mirbsd*)
+    DLL_SUFFIX=".so.${SO_VERSION}"
+    DSO_CFLAGS=''
+    DSO_PIC_CFLAGS='-fPIC'
+    DSO_LDOPTS='-shared -fPIC'
+    if test "$LIBRUNPATH"; then
 	DSO_LDOPTS="-R$LIBRUNPATH $DSO_LDOPTS"
     fi
     ;;
@@ -3231,7 +3241,7 @@ then
 			fi
 			;;
 
-	    *-*-openbsd*|*-*-bsdi*)
+	    *-*-openbsd*|*-*-bsdi*|*-*-midnightbsd*|*-*-mirbsd*)
 			AC_DEFINE(_REENTRANT)
 			AC_DEFINE(_THREAD_SAFE)
 			dnl -pthread links in -lc_r, so don't specify it explicitly.
